consumers/writers/cassandra/init.go

```diff
@@ -31,6 +31,6 @@ const (
         protocol text,
         created bigint,
         payload text,
-        PRIMARY KEY (channel, created, id)
+        PRIMARY KEY (channel, created, id, publisher)
```

Comment:
@AryanGodara , Please discard this changes , since we are not touching cassandra